src/main/java/com/ea/restaurant/entities/BaseEntity.java

```diff
@@ -1,6 +1,11 @@
 package com.ea.restaurant.entities;
 
 import com.ea.restaurant.constants.Status;
+import jakarta.annotation.Nonnull;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.MappedSuperclass;
```

Comment:
This are all coming from jakarta, are you sure this is ok? Don't you prefer to use JPA `javax.persistance....`?

Reply:
Java Persistence has become Jakarta Persistence as part of the overall move of Java EE to Jakarta. Various legal requirements forced the changing of the javax namespace - for persistence, that means changing from javax.persistence to jakarta.persistence for package names as well as property and hint names.